ListOfSobjectsTest.cls

```diff
@@ -0,0 +1,28 @@
+@IsTest
+public class ListOfSobjectsTest {
+    @IsTest static void testStandardObjectsInOrg(){
+        List<String> ourTestObjects = new List<String>{'Account','Contact'};
+        List<String> sobjectNameList = sObjectListInOrg.getsObjectListInOrg(); 
+        for(String expectedSobject : ourTestObjects){
+            if(sobjectNameList.Contains(expectedSobject)){

```

Comment:
Can we write link this : - 
for(String expectedSobject : ourTestObjects){ 
System.assert(sobjectNameList.Contains(expectedSobject), expectedSobject + 'object not found in org');
}